packages/static-wado-creator/lib/operation/InstanceDeduplicate.js

```diff
@@ -16,7 +16,7 @@ const extractors = {
  * and then collects the deduplicate data into a list.
  * Whenever the study UID changes, a new event is fired to indicate the deduplicated data is ready.
  */
-async function deduplicateSingleInstance(id, imageFrame) {
+async function deduplicateSingleInstance(id, imageFrame, options) {
```

Comment:
Generaly options is passed into a factory method which creates the functions that handle the various events.
You can leave this one, but that is the usual design, as it allows validation of options etc right up front.

Reply:
Ok, the factory method was already there for the bulkdata, but the options weren't actually being used - so I misunderstood why they didn't need to be passed along.